tools/laravel.md

```diff
@@ -9,17 +9,23 @@ command: composer show laravel/framework|grep versions
 releaseDateColumn: true
 sortReleasesBy: 'releaseCycle'
 releases:
+  - releaseCycle: "8.x"
+    release: 2020-09-08
+    support: 2021-03-08
+    eol: 2022-09-08
```

Comment:
According to https://laravel.com/docs/8.x/releases#support-policy it should be `2021-09-08`. Do you agree?

Reply:
yes updated the PR :-)